src/EnrollWebauthnController.js

```diff
@@ -134,32 +134,38 @@ function (Okta, Errors, FormType, FormController, CryptoUtil, webauthn, Footer,
         var children = [];
 
         if (webauthn.isNewApiAvailable()) {
-          //enroll.webauthn.instructions.edge is unescaped because it contains html
+          //enroll.webauthn.v2.instructions.edge is unescaped because it contains html
           children.push(FormType.View({
             View: Okta.View.extend({
               className: 'webauthn-enroll-text',
               template: '\
                 <div class="webauthn-enroll-instructions">\
-                  <p>{{i18n code="enroll.webauthn.instructions" bundle="login"}}</p>\
+                  <p>{{i18n code="enroll.webauthn.v2.instructions" bundle="login"}}</p>\
                 </div>\
                 {{#if isEdge}}\
                   <div class="webauthn-edge-text">\
-                     <p>{{{i18n code="enroll.webauthn.instructions.edge" bundle="login"}}}</p>\
+                     <p>{{{i18n code="enroll.webauthn.v2.instructions.edge" bundle="login"}}}</p>\
+                  </div>\
+                {{/if}}\
+                {{#if onlySupportsSecurityKey}}\
+                  <div class="webauthn-restrictions-text">\
+                     <p>{{{i18n code="enroll.webauthn.instructions.noSupportForBiometric" bundle="login"}}}</p>\
                   </div>\
                 {{/if}}\
                 <div data-se="webauthn-waiting" class="okta-waiting-spinner hide"></div>\
               ',
               getTemplateData: function () {
                 return {
-                  isEdge: BrowserFeatures.isEdge
+                  isEdge: BrowserFeatures.isEdge,
+                  onlySupportsSecurityKey: (BrowserFeatures.isFirefox || BrowserFeatures.isSafari) && (BrowserFeatures.isMac)
```

Comment:
Are there any other browser / OS (including devices) combination that might not support biometric? Would it make sense to show this note in all cases?

Reply:
windows - edge, chrome and firefox support both 
mac - only chrome support both 
android - chrome supports both I guess 
ios - no webauthn support 

I guess mac firefox/safari seems to be the most common that we would like to check for and show a message.